cf-execd/cf-execd.c

```diff
@@ -424,6 +424,17 @@ void ThisAgentInit(void)
     umask(077);
 }
 
+// Return 2 to 62 seconds depending on how much time is left until the next
+// minute starts. Since cf-execd wakes up "every minute" to evaluate its
+// schedule, we want to do this at the start of the minute, to avoid
+// accidentally skipping any runs if things are slow.
+static inline time_t GetPulseTime()
+{
+    const time_t current_second = (time(NULL) % 60);
+    const time_t remaining_seconds = 60 - current_second;
```

Comment:
```suggestion
    const time_t remaining_seconds = CFPULSETIME - current_second;
```

Reply:
but this is the cherry pick, so change should go in https://github.com/cfengine/core/pull/5498

Reply:
I suggested in the main PR to just remove `CFPULSETIME` https://github.com/cfengine/core/pull/5498#discussion_r1592703126